Makefile.am

```diff
@@ -295,6 +296,67 @@ libeos_updater_flatpak_installer_libeos_updater_flatpak_installer_@EUFI_API_VERS
 # eos-updater-flatpak-installer program
 libexec_PROGRAMS += eos-updater-flatpak-installer/eos-updater-flatpak-installer
 
+# Gobject Introspection
+-include $(INTROSPECTION_MAKEFILE)
+INTROSPECTION_GIRS =
+INTROSPECTION_SCANNER_ARGS =
+INTROSPECTION_COMPILER_ARGS =
+AM_DISTCHECK_CONFIGURE_FLAGS += --enable-introspection
+EXTRA_DIST += m4/introspection.m4
+
+if HAVE_INTROSPECTION
+# Gobject Introspection for libeos-updater-util
+libeos-updater-util/EosUpdaterUtil-@EUU_API_VERSION@.gir: libeos-updater-util/libeos-updater-util-@EUU_API_VERSION@.la
+libeos_updater_util_EosUpdaterUtil_@EUU_API_VERSION@_gir_INCLUDES = GObject-2.0 Gio-2.0
+libeos_updater_util_EosUpdaterUtil_@EUU_API_VERSION@_gir_CFLAGS = \
+	$(libeos_updater_util_libeos_updater_util_@EUU_API_VERSION@_la_CFLAGS)
+libeos_updater_util_EosUpdaterUtil_@EUU_API_VERSION@_gir_FILES = \
+	$(libeos_updater_util_libeos_updater_util_@EUU_API_VERSION@_la_SOURCES)
+libeos_updater_util_EosUpdaterUtil_@EUU_API_VERSION@_gir_LIBS = libeos-updater-util/libeos-updater-util-@EUU_API_VERSION@.la
+libeos_updater_util_EosUpdaterUtil_@EUU_API_VERSION@_gir_NAMESPACE = EosUpdaterUtil
+libeos_updater_util_EosUpdaterUtil_@EUU_API_VERSION@_gir_SCANNERFLAGS = \
+	$(WARN_SCANNERFLAGS) \
+	--nsversion=@EUU_API_VERSION@ \
+	--c-include="libeos-updater-util/util.h" \
+	--identifier-prefix=EosUpdater \
+	--identifier-prefix=Euu \
+	--symbol-prefix=euu \
+	-I$(top_srcdir) \
```

Comment:
I’d drop the `-I` here and add `$(libeos_updater_util_libeos_updater_util_@EUU_API_VERSION@_la_CPPFLAGS)` to `libeos_updater_util_EosUpdaterUtil_@EUU_API_VERSION@_gir_CFLAGS`. Ensures they’re built with the same set of preprocessor flags.